pbdf/Issues/iban/description.xml

```diff
@@ -0,0 +1,80 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Bank account</en>
+		<nl>Bankrekening</nl>
+	</Name>
+	<ShortName>
+		<en>Bank</en>
+		<nl>Bank</nl>
+	</ShortName>
+	<SchemeManager>pbdf-staging</SchemeManager>
+	<IssuerID>pbdf</IssuerID>
+	<CredentialID>iban</CredentialID>
+	<Description>
+		<en>Your account details, provided by your bank, obtained through a token payment</en>
```

Comment:
"obtained through a token payment" is maybe a bit formal. Perhaps "small payment" is better.